pkg/scenarioserver/scenarioserver.go

```diff
@@ -697,6 +698,18 @@ func (s ScenarioServer) CreateFunc(w http.ResponseWriter, r *http.Request) {
 	scenario.Spec.Tags = tags
 	scenario.Spec.KeepAliveDuration = keepaliveDuration
 
+	rawVMTasks := r.PostFormValue("vm_tasks")
+	if rawVMTasks != "" {
```

Comment:
in the Update Metod you validate the Input, here you do not validate the input. Better to build a function to valdiate VirtualMachineTasks and use it in both methods